src/de/mod.rs

```diff
@@ -387,6 +396,72 @@ where
         Self::new(reader)
     }
 
+    /// Set the maximum number of events that could be skipped during deserialization
+    /// of sequences.
+    ///
+    /// If `<element>` contains more than specified nested elements, `#text` or
+    /// CDATA nodes, then [`DeError::TooManyEvents`] will be returned during
+    /// deserialization of sequence field (any type that uses [`deserialize_seq`]
+    /// for the deserialization, for example, `Vec<T>`).
+    ///
+    /// This method can be used to prevent a [DoS] attack and infinity memory
+    /// consumption when parse a very large or even an infinity XML to a sequence
```

Comment:
```suggestion
    /// consumption when parsing a very large or even an infinite XML to a sequence
```

Could you explain "infinite xml"?  (to me, not in the text)

Reply:
I mean, that someone could try to parse an XML stream what will never end, for example, some live event log:
```xml
<EventList>
  <Event/>
  <Event/>
  <Event/>
  ...
```
But without deserializing such stream in a streaming way it anyway impossible to get full list regardless of this option. I'll remove that sentence